src/k8s/pkg/k8sd/setup/k8s_apiserver_proxy.go

```diff
@@ -17,10 +17,15 @@ func K8sAPIServerProxy(snap snap.Snap, servers []string, extraArgs map[string]*s
 		return fmt.Errorf("failed to write proxy configuration file: %w", err)
 	}
 
+	localhostAddress := "127.0.0.1"
+	if len(servers) > 0 && !utils.IsIPv4(servers[0]) {
```

Comment:
is it possible to have a mix of ipv4 and v6?

Reply:
let's remove this from here altogether and get the address from the bootstrap hook.